lib/data/marriage_abroad_services.yml

```diff
@@ -575,6 +575,17 @@ romania:
       - :receiving_notice_of_marriage
       - :issuing_cni_or_nulla_osta
   payment_partial_name: pay_by_cash_or_credit_card_no_cheque
+  same_sex:
+    default:
+      partner_british:
+      - :receiving_notice_of_registration
+      - :registering_marriage
+      - :issuing_marriage_certificate
+      partner_other:
+      - :receiving_notice_of_registration
+      - :registering_marriage
+      - :issuing_marriage_certificate
+  payment_partial_name: pay_by_cash_or_credit_card_no_cheque
```

Comment:
I think it's probably best to keep them separate, unless we know that these services should also be shown for same sex ceremonies in Romania when your partner is Romanian.

I know I haven't done a very good job of explaining the purpose of marriage_abroad_services.yml but my hope is that it'll allow us to always render the service/fees partials in _every_ outcome. At that point, it'll be the data in this file that dictates whether or not a service/fee is shown, rather than a combination of this data and logic in the template.